### PR TITLE
 Test pattern selection for system role 'kvm host' (fate#317481)

### PR DESCRIPTION
**Pull request by imobach**
Superesedes #1424 adding updates from code review.

### CODE REVIEW

File: tests/console/patterns.pm [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2016 SUSE LLC
```
Comment (coolo) [inline tests/console/patterns.pm]: there is still a copyright for 2009, please remove every year but 2016

**Comment by okurz** [comment]
after incorporating that change please squash the "Updates from code review" commit into the previous one.

**Comment by imobach** [comment]
Sorry (I overlooked 2009 stuff). It should be fixed (and squashed) now.

Thanks!

**Comment by okurz** [comment]
thanks for the contribution. Now this needs to be scheduled in a test_suite. who can help me with that?

**Comment by imobach** [comment]
Just for the records, progress will be [tracked here](https://progress.opensuse.org/issues/12224).

**Pull request merged**